docs/develop/overview-authn-authz.md

```diff
@@ -45,7 +45,7 @@ These samples, however, do not use the token as an ID token. They use it to get
 
 ## Access to Microsoft Graph with SSO
 
-To use SSO to get access to Microsoft Graph, your add-in in a task pane or function file calls the [getAccessToken](/javascript/api/office/officeruntime.auth#getAccessToken-options--callback-) method. If the user is not signed into Office, Office will open a dialog and navigate it to the Azure Active Directory login page. After the user is signed in, or if the user is already signed in, the method returns an access token. The token is a bootstrap token in the **On Behalf Of** flow. Specifically, it has a `scope` claim with the value `access_as_user`. For guidance about the claims in the token, see [Microsoft identity platform access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens#payload-claims). For an example of a one of these tokens, see the [Example access token](sso-in-office-add-ins.md#example-access-token).
+To use SSO to get access to Microsoft Graph, your add-in in a task pane or function file calls the [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-) method. If the user is not signed into Office, Office will open a dialog and navigate it to the Azure Active Directory login page. After the user is signed in, or if the user is already signed in, the method returns an access token. The token is a bootstrap token in the **On Behalf Of** flow. Specifically, it has a `scope` claim with the value `access_as_user`. For guidance about the claims in the token, see [Microsoft identity platform access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens#payload-claims). For an example of a one of these tokens, see the [Example access token](sso-in-office-add-ins.md#example-access-token).
```

Comment:
The bookmark portion of the link target `/javascript/api/office-runtime/officeruntime.auth#getAccessToken-options-)` needs to be all lowercase in order for the bookmark to work properly.

i.e., it should be:   `/javascript/api/office-runtime/officeruntime.auth#getaccesstoken-options-)`